modules/apps/portal-settings/portal-settings-authentication-ldap-web/src/main/java/com/liferay/portal/settings/authentication/ldap/web/internal/portlet/action/EditLDAPServerMVCActionCommand.java

```diff
@@ -302,18 +256,34 @@ private void _validateSearchFilters(ActionRequest actionRequest)
 		_ldapFilterValidator.validate(groupFilter, "importGroupSearchFilter");
 	}
 
-	private static ConfigurationProvider<LDAPServerConfiguration>
-		_ldapServerConfigurationProvider;
-
+	@Reference
 	private CounterLocalService _counterLocalService;
 
 	@Reference
 	private LDAPFilterValidator _ldapFilterValidator;
 
+	@Reference(
+		target = "(factoryPid=com.liferay.portal.security.ldap.configuration.LDAPServerConfiguration)"
+	)
+	private ConfigurationProvider<LDAPServerConfiguration>
+		_ldapServerConfigurationProvider;
+
+	@Reference
 	private Portal _portal;
+
+	@Reference(
+		target = "(javax.portlet.name=" + ConfigurationAdminPortletKeys.INSTANCE_SETTINGS + ")"
+	)
 	private Portlet _portlet;
+
 	private PortletContext _portletContext;
```

Comment:
@slnn This was added by LPS-61991, but the usage was soon removed by LPS-67555. Please remove this field, `_portletContextFactory`, and other fields if possible.